packages/devtools_app/lib/src/memory/flutter/memory_controller.dart

```diff
@@ -135,6 +226,8 @@ class MemoryController extends DisposableController
 
   bool hasStopped;
 
+  VM _vm;
```

Comment:
do we need to store this locally? can we just access it through serviceManager instead?

Reply:
I suppose but that's an await every time I build the Android Memory button I guess I could cache the notion of Android device.  Seem more efficient.